### ISSUE TITLE
Did not work for us :(

### ISSUE THREAD

**Issue by i5513** [mattn/vim-metarw-redmine#1]
Hello,

We have redmine 3.0.4, From the project list, always the same project tasks are showed. Not matter which project you choose, always the same project task is shown.

I'm not sure if it is a working project (for multiproject redmine instances). I would like to have time to improve ...

Thank you ! A useful idea !

**Comment by mattn** [comment]
How didn't work? please describe more details.

**Comment by mattn** [comment]
added `gom update`. try last commit. But note that this feature is experimental. Please try it after backup your files.

**Comment by i5513** [comment]
what is gom update ? I have to install something go related ? 

See: 

> metarw content browser
> redmine:
> 
> first/
> second/
> ...
> nth/

If I type 'o' on "first" I get:

> first task
> second task

If I type 'o' on the second project I get the same list of tasks

maybe I have to install something more?

My config is with NeoBundler, this is the snipper where I config metarw-redmine:

> NeoBundle 'kana/vim-metarw'
> NeoBundle 'mattn/webapi-vim'
> NeoBundle 'mattn/vim-metarw-redmine'
> let g:metarw_redmine_server="http://xxx/
> let g:metarw_redmine_apikey="e2fefc1dfakldfjadafdafa"

Thank you very much !
PD: I'm tempted to do a tcdump to see which url is wrong formed, but I'm not know enough viml to fix it (I think)

**Comment by mattn** [comment]
Sorry. previous comment was my mistake. So removed.

**Comment by mattn** [comment]
Are you on OSX or Linux ?

**Comment by i5513** [comment]
Yes, I'm on linux, is it a problem ? :(

**Comment by mattn** [comment]
> Yes, I'm on linux, is it a problem ? :(

No, but if you use windows, you should use https://github.com/mattn/vim-metarw instead.

What the errors did you get?

**Comment by i5513** [comment]
I don't see any error, are there any log? 

I can switch to mattn/vim-metarw if required.  Thank you !

**Comment by mattn** [comment]
Currently, don't have any logs. Sorry. And no need to switch if you are not on windows.
Hmm, you can see the list of project, but can't see issue list. Right?

**Comment by i5513** [comment]
The issue list did not correspond with the project that I choose, always the same issue list is showed. On wireshark ,  I can see : 

http://myhost//issues.json?key=mykey

Our redmine have a hierarchical list of project (all projects are sons of the another project, except the grand father project)

Maybe that is the issue

**Comment by i5513** [comment]
I have tested with a redmine with some projects. I can see all the issues, but not selected by projects. Would be nice if you can select the project and see only its issues (using ?project_id and maybe subproject_id)

Thank you!
